test/postgresql.migration.test.js

```diff
@@ -73,6 +73,42 @@ describe('migrations', function() {
       done();
     });
   });
+
+  it('OrderData should have correct prop type uuid with custom generation function', function(done) {
+    checkColumns('OrderData', function(err, cols) {
+      assert.deepEqual(cols, {
+        ordercode:
+        {column_name: 'ordercode',
+          column_default: 'uuid_generate_v1()',
+          data_type: 'uuid'},
+        ordername:
+        {column_name: 'ordername',
+          column_default: null,
+          data_type: 'text'},
+        id:
+        {column_name: 'id',
+          column_default: 'nextval(\'orderdata_id_seq\'::regclass)',
+          data_type: 'integer'},
+      });
+      done();
+    });
+  });
+
+  it('DefaultUuid should have correct id type uuid and default function v4', function(done) {
+    checkColumns('DefaultUuid', function(err, cols) {
+      assert.deepEqual(cols, {
+        defaultcode:
+        {column_name: 'defaultcode',
+          column_default: 'uuid_generate_v4()',
```

Comment:
IIUC the `column_default` here is `uuid_generate_v4()` instead of `uuid_generate_v1()` because `extension` is missing in the [property def](https://github.com/strongloop/loopback-connector-postgresql/pull/404/files?utf8=%E2%9C%93&diff=split&w=1#diff-a55dd5264ec27c23b0c70c696af975eeR171), so I am thinking, maybe add a warning before [line 318](https://github.com/strongloop/loopback-connector-postgresql/pull/404/files?utf8=%E2%9C%93&diff=split&w=1#diff-546064e86fc7d116e6952283c43b8069R318) to let people know it falls back to the default value due to missing extension?

Reply:
Correct, it should  use the default extension and function for this case. And I have `uuid_generate_v4()`b here :smile: